motoman_gp180_support/urdf/gp180_macro.xacro

```diff
@@ -33,64 +33,74 @@
     </link>
     <link name="${prefix}link_2_l">
         <visual>
+            <origin rpy="${radians(90)} ${radians(0)} ${radians(90)}" />
             <geometry>
                 <mesh filename="package://motoman_gp180_support/meshes/gp180_120/visual/link_2_l.stl"/>
             </geometry>
             <xacro:material_yaskawa_blue/>
         </visual>
         <collision>
+            <origin rpy="${radians(90)} ${radians(0)} ${radians(90)}" />
             <geometry>
                 <mesh filename="package://motoman_gp180_support/meshes/gp180_120/collision/link_2_l.stl"/>
             </geometry>
         </collision>
     </link>
     <link name="${prefix}link_3_u">
         <visual>
+            <origin rpy="${radians(-90)} ${radians(0)} ${radians(-90)}" />
             <geometry>
                 <mesh filename="package://motoman_gp180_support/meshes/gp180/visual/link_3_u.stl"/>
             </geometry>
             <xacro:material_yaskawa_blue/>
         </visual>
         <collision>
+            <origin rpy="${radians(-90)} ${radians(0)} ${radians(-90)}" />
             <geometry>
                 <mesh filename="package://motoman_gp180_support/meshes/gp180/collision/link_3_u.stl"/>
             </geometry>
         </collision>
     </link>
     <link name="${prefix}link_4_r">
         <visual>
+            <origin rpy="${radians(0)} ${radians(90)} ${radians(0)}" xyz="0 0 1.225"/>
```

Comment:
Is the `xyz` part a typo? It's not specified for the collision object or any of the other links.